wp-content/themes/core/assets/css/src/theme/spacing/sink.pcss

```diff
@@ -0,0 +1,153 @@
+ *
+ * This class should be used as sparingly as possible.
+ *
+ * ------------------------------------------------------------------------- */
+
+.s-sink {
+	/* -----------------------------------------------------------------------------
+	 * First & last element overrides
+	 * ----------------------------------------------------------------------------- */
+
+	& > {
+		/* CASE: remove top margin from first element */
+		*:first-child {
+			margin-top: 0;
+		}
+
+		/* CASE: remove bottom margin from bottom element */
+		*:last-child {
+			margin-bottom: 0;
+		}
+	}
+
+	/* -----------------------------------------------------------------------------
+	 * Headings
+	 * ----------------------------------------------------------------------------- */
+
+	h1,
+	.h1,
+	h2,
+	.h2,
+	h3,
+	.h3,
+	h4,
+	.h4,
+	h5,
+	.h5,
+	h6,
+	.h6 {
+		margin-top: var(--spacer-60);
+		margin-bottom: var(--spacer-20);
+	}
+
+	/* -----------------------------------------------------------------------------
+	 * Horizontal Rule
+	 * ----------------------------------------------------------------------------- */
+
+	hr {
+		margin-top: var(--spacer-60);
+		margin-bottom: var(--spacer-60);
+	}
+
+	/* -------------------------------------------------------------------------
+	 * Lists
+	 * ------------------------------------------------------------------------- */
+
+	/* Shared */
+	ul,
+	.list-bullet,
+	ol,
+	.list-number,
+	dl,
+	.dl {
+		margin-top: var(--spacer-40);
+		margin-bottom: var(--spacer-40);
+	}
+
+	/* List Item */
+	li {
+
+		& ~ li {
+			margin-top: var(--spacer-20);
+		}
+	}
+
+	/* Definition */
+	dd,
+	.dd {
+		margin-bottom: var(--spacer-40);
+	}
+
+	/* -----------------------------------------------------------------------------
+	 * Paragraph
+	 * ----------------------------------------------------------------------------- */
+
+	p {
+		margin-bottom: var(--spacer-40);
+	}
+
+	/* -----------------------------------------------------------------------------
+	 * Quote & Blockquote
+	 * ----------------------------------------------------------------------------- */
+
+	/* Shared */
+	blockquote,
+	.blockquote,
+	q,
+	.quote {
+		margin: var(--spacer-60) auto;
+	}
+
+	/* q */
+	q,
+	.quote {}
+
+	/* blockquote */
+	blockquote,
+	.blockquote {
+
+		p {
+			margin: 0 auto;
+
+			& ~ p {
+				margin-top: var(--spacer-40);
+			}
+		}
+	}
+
+	/* cite */
+	cite,
+	.cite {
+		display: block;
+		margin-top: var(--spacer-10);
+	}
+
+	/* -----------------------------------------------------------------------------
+	 * Table
+	 * ----------------------------------------------------------------------------- */
+
+	table,
+	.table {
+		margin: var(--spacer-60) auto;
+	}
+
+	/* -----------------------------------------------------------------------------
+	 * Overline
+	 *
+	 * Overrides default selector spacing.
+	 * ----------------------------------------------------------------------------- */
+
+	.t-overline {
```

Comment:
What is this, different from .h1-h6?

Reply:
It's a style that can be applied to any element, such as a `p` that appears before a heading: http://p.tri.be/Zqe47q

Reply:
Got it, does that then also mean that all the other context named typo helper classes should also get added to here?

Reply:
They are already available via this sink spacing file and the typography sink styles (as is this typographic style). This code is just to handle an edge case/exception for spacing via the sibling selector.

Reply:
Talking about all of these `.t-display-x-large`, similar to `.t-overline`, it doesn't look like we are applying any vertical spacing to them by default when wrapped in our vert spacing helper. Confirming that is intentional and gtg?

Reply:
Perhaps we could key in on some global defaults with attribute selectors `[class*="t-display"]` and do by groupings

Reply:
Confirmed. That is intentional and g2g. From the designs, the overline style is unique in that it can be applied to other elements to produce a different spacing flow than the other blocks/tags. I don't think we want to apply default sink spacing to all the typographic helpers out of the box. That would land us back in the over-use of `t-content` scenario where _all_ the typograhic styles have automatic spacing which needs to be overridden.